cmd/identityoracled/bootstrap/bootstrap.go

```diff
@@ -115,7 +116,22 @@ func Run(approver oracle.ApproverInterface) {
 	}
 
 	spyNode := spynode.NewNode(spyConfig, spyStorage)
-	spyNode.AddTxFilter(&TxFilter{})
+
+	// -------------------------------------------------------------------------
```

Comment:
Let's lose these `/ -----------------------....` lines

If these setup is that big, use funcs rather than comments to clarify the setup.

Reply:
That will make the code more complex and less readable. The functions would each require a fair number of parameters and would greatly increase the line count and code complexity. I don't see any benefit.